index.php

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset = "UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Practitioner Login</title>
+    <link href="login_layout.css" rel="stylesheet" type="text/css">
+</head>
+<center>
+    <h1> Medication and Diet Regime Management System </h1>
+    <body bgcolor="#E7FBFC">
+        <div>
+            <h2>
+                Login
+            </h2>
+        </div>
+        
+        <form action="login.php" method="POST">
+            <div class="container">
+                <label><b>Email</b></lable>
```

Comment:
label is spelled wrong

Reply:
i just changed it HAHAHA, spotted it at the same time